### ISSUE TITLE
Add arn as an attribute for aws_launch_template

### ISSUE THREAD

**Issue by tdmalone** [terraform-providers/terraform-provider-aws#5028]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```
Terraform v0.11.7
+ provider.aws v1.25.0
```

### Affected Resource(s)

* [aws_launch_template](https://www.terraform.io/docs/providers/aws/r/launch_template.html)

### Terraform Configuration Files

```hcl
# Ideal.
output "arn" {
  value = "${aws_launch_template.main.arn}"
}

# Current workaround.
output "arn" {
  value = "arn:aws:ec2:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:launch-template/${aws_launch_template.main.id}"
}
```

### Expected Behavior

Expected (or hoped) there would be an ARN attribute exported for the launch template ;)

### Actual Behavior

```
Error: Error running plan: 1 error(s) occurred:

* output.arn: Resource 'aws_launch_template.main' does not have attribute 'arn' for variable 'aws_launch_template.main.arn'
```

### Important Factoids

- Similar to #5027

**Comment by MattMcNam** [comment]
Interested in working on this in the next day or so

**Comment by bflad** [comment]
This has been released in [version 1.29.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1290-july-26-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
